using-valtimo/form-link/configure-task.md

```diff
@@ -1,14 +1,14 @@
 # How to link a form to a task
 
 There are two ways to connect a form to a task, with a preliminary step to ensure the form can be
-found. What this preliminary step is depends on the type of form that should be linked to a task.
+found. What this preliminary step is, depends on the type of form that should be linked to a task.
 
 ## Configuring a form link through the UI
 
 To create a form link for a start event or user task, the following steps are necessary:
 
-1. In the admin menu, go to "Process links". This shows a  select box, specifying which process and
-   version is being configured, as well as a diagram for the current selection.
+1. In the menu 'Admin', go to "Process links". This shows a select box, specifying which process and
```

Comment:
I agree with the quotes here, but I don't know if I agree with switching the order of admin and menu. To me that reads a bit less natural, not sure what others think.